pallets/polkadex-ido/src/lib.rs

```diff
@@ -674,13 +672,11 @@ pub mod pallet {
         ///
         /// * `round_id`: Funding round id
         #[pallet::weight((10_000, DispatchClass::Normal))]
-        pub fn show_interest_in_round(origin: OriginFor<T>, round_id: T::Hash, amount: BalanceOf<T>) -> DispatchResult {
+        pub fn invest(origin: OriginFor<T>, round_id: T::Hash, amount: BalanceOf<T>) -> DispatchResult {
```

Comment:
I assume description should be properly updated if we changing the logic and basically removing old method and adding new one (parameters should be updated as well).

Reply:
1) High level docblock should be updated as well [here](https://github.com/Polkadex-Substrate/Polkadex/blob/3100a68a3324a65f448c42ec67b66cdf0aabba8c/pallets/polkadex-ido/src/lib.rs#L53).
2) [Readme](https://github.com/Polkadex-Substrate/Polkadex/blob/3100a68a3324a65f448c42ec67b66cdf0aabba8c/pallets/polkadex-ido/README.md?plain=1#L84) should be updated as well.

Reply:
Updated ✅